### PR TITLE
Fix typos 

### PR DESCRIPTION
**Pull request by Blackjacx**
### Checklist

- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I see several green `ci/circleci` builds in the "All checks have passed" section of my PR ([connect CircleCI to GitHub](https://support.circleci.com/hc/en-us/articles/360008097173-Why-aren-t-pull-requests-triggering-jobs-on-my-organization-) if not)
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.
- [x] I've added or updated relevant unit tests.

### Motivation and Context

It fixes typos in the output of `fastlane action ...`

### Description

It fixes typos in the output of `fastlane action ...` for the commands snapshot, scan and gym

### CODE REVIEW

File: scan/lib/scan/options.rb [hunk 1]
```diff
@@ -31,7 +31,6 @@ def self.available_options
                                      optional: true,
                                      env_name: "SCAN_PROJECT",
                                      description: "Path to the project file",
-                                     conflicting_options: [:package_path],
```
Comment (lacostej) [inline scan/lib/scan/options.rb:34]: Hello, thanks for the contribution. Why remove this line?

Comment (rogerluan) [reply scan/lib/scan/options.rb:34]: ☝️

**Review by rogerluan — COMMENTED** [review]
LGTM except for the comment above 🙏 
Also: hi @Blackjacx 👋 😁  long time no see!

**Comment by lacostej** [comment]
@Blackjacx Thanks for the contribution. I allowed myself to incorporate the review comments into your code. Let's merge this!

**Review by lacostej — APPROVED** [review]

**Comment by Blackjacx** [comment]
Oh Hey there 👋 thanks for dealing with my PR. I was very short on time and did not have  alook in here for a long time. Sorry for that.

And hello @rogerluan - nice to meet you again 👋

**Pull request merged**